core/arch/arm/tee/arch_svc.c

```diff
@@ -134,6 +135,7 @@ static const struct syscall_entry tee_svc_syscall_table[] = {
 	SYSCALL_ENTRY(syscall_se_channel_transmit),
 	SYSCALL_ENTRY(syscall_se_channel_close),
 	SYSCALL_ENTRY(syscall_cache_operation),
+	SYSCALL_ENTRY(syscall_gprof_send),
```

Comment:
What's the other option(s)? A static TA? Would be OK if it were as simple to implement and invoke from `libutee` as a syscall. Is it?